src/Templates/src/templates/maui-blazor/Routes.razor

```diff
@@ -0,0 +1,6 @@
+﻿<Router AppAssembly="@typeof(MauiProgram).Assembly">
```

Comment:
I think you still want the NotFound content here. It's not used in a Blazor Web App, but I think it can still show up in a Blazor Hybrid up if you mistype a link to a page.

Reply:
I chatted with @danroth27 and we agreed it's OK to have this screen. Ultimately if anyone ever sees this then it's a bug in the app, so it doesn't matter much what it looks like. Unlike Blazor Web apps you don't get to type into an address bar to go wherever you want, so you should never see this.

Reply:
Agreed for now.

Note that this reasoning may cease to apply in the future if we implement https://github.com/dotnet/aspnetcore/issues/45654, which would let people programmatically trigger the `NotFound` content, e.g., if you visited a link to `/entries/{id}` and then at runtime the DB/API backend says there is no such ID.